redirects.yaml

```diff
@@ -296,7 +296,6 @@ desktop/take-the-tour/?: "http://tour.ubuntu.com/"
 desktop/tenders/?: "/desktop/contact-us"
 desktop/tenders/contact-us/?: "/desktop/contact-us"
 desktop/tenders/thank-you/?: "/desktop/thank-you"
-desktop/ubuntu-kylin.*: "https://cn.ubuntu.com/desktop/"
```

Comment:
I'm not sure but I think it could be worth leaving these redirects in, in case they're referred to on other sites.